setup.py

```diff
@@ -75,18 +90,21 @@ def package_files(directory):
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
     ],
     python_requires=">=3.6",
     # What does your project relate to?
     keywords="pca report automation",
-    packages=find_packages(where="src"),
-    package_dir={"": "src"},
+    packages=find_packages(where="src/pca_report_library"),
+    package_dir={"": "src/pca_report_library"},
```

Comment:
This breaks your package namespace.

```suggestion
    packages=find_packages(where="src"),
    package_dir={"": "src"},
```

Reply:
Resolved in a1252af